fields.yml

```diff
@@ -1335,3 +1335,42 @@
           description: >
             Group the user is a part of. This field can contain a list of groups, if
             necessary.
+    
+    - name: user_agent
+      title: User agent
+      group: 2
+      description: >
+        The user_agent fields normally come from a browser request. They often
+        show up in web service logs coming from the parsed user agent string.
+      type: group
+      fields:
+    
+        - name: original
+          level: extended
+          type: keyword
+          description: >
+            Unparsed version of the user_agent.
+    
+        - name: name
+          level: extended
+          type: keyword
+          example: Chrome
+          description: >
+            Name of the user agent.
+        - name: version
+          level: extended
+          type: keyword
+          description: >
+            Version of the user agent.
+    
+        - name: device.name
```

Comment:
How does this correlate with https://github.com/elastic/ecs/pull/238 ?

Reply:
It correlates in absolutely no way. Here `user_agent.device.name` would contain things like "iPad", "iPhone"...